### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SCREENREADER, COGNITION]: Collectively major structure, heading, and organization issues on the main page "Locations section"

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#25172]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

## User Story or Problem Statement:
As a screen reader user, I expect web pages to be organized logically. I expect to determine based on HTML elements how content is related, and I expect to determine a link's purpose and target by its link text.

## Details
The Locations section has several issues with structure, organization, etc. which cumulatively make it impossible for a screen reader user to interpret content.
- Under the level 3 heading "Main Location," the headings "Hour" and "Main Line need to be level 4 headings. 
- "Other locations" needs to be a level 3 heading located below "Hours." As the page is structured, the implication is that the "Hours" section refers to the satellite locations. 

_Michelle's edits - 
<details>
<summary> "Location section" screenshot for reference </summary> 

![image](https://user-images.githubusercontent.com/55411834/119402134-bb1a1900-bc99-11eb-8c56-efc355e909f4.png)

</details>

## Acceptance Criteria 
- [ ] Subheadings under "main location" are tagged as level 4 headings.
- [ ] Locations section is organized such that it is easy to determine which subheadings are under "main location" and which are under "Other locations."

_Michelle's additional AC following grooming:_
- [ ] The Location section on the Main page is organized with the appropriate headings as follows:

> Main location - H3
> Direct line - H4
> Hours - H4
> Alternate hours statement
> Other locations - H3

**See Leyda's updated ["MVP mobile - main" in UXPin](https://preview.uxpin.com/0a5eaacdf411fde0cb3abf4932010c29fbccb7de#/pages/138920563)**

**Comment by mmiddaugh** [comment]
Separated this area of concern out of #25019

**Comment by lildziner** [comment]
I updated ["MVP mobile - main" in UXPin](https://preview.uxpin.com/0a5eaacdf411fde0cb3abf4932010c29fbccb7de#/pages/138920563) to reflect the new order and heading level changes. This version of the prototype also reflects the design and language changes made to the alerts in response to staging review design feedback [#24791](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24791#issuecomment-849013169). **Note:** Unfortunately, this prototype is not accessible.


![image.png](https://images.zenhubusercontent.com/5c4895778d88074ab2c6ad18/d805e340-a725-4a83-ad0d-c75b5567d097)

**Comment by mmiddaugh** [comment]
Please see my [comments in the staging review ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24791#issuecomment-849966580) regarding the source of the language currently used

**Comment by mmiddaugh** [comment]
Will have @AngelaFowler82 validate this issue once we have a Vet Center published and available on Staging.

**Comment by mpelzsherman** [comment]
Can we use either https://www.va.gov/colorado-springs-vet-center/ or https://www.va.gov/nashville-vet-center/ to validate?

**Comment by joshkimux** [comment]
<img width="1318" alt="Screen Shot 2021-07-16 at 11 43 31 AM" src="https://user-images.githubusercontent.com/14154792/125974050-0426955c-6247-44a7-82e1-0c985719260a.png">

Opening up headingsMap it looks like this has been resolved. @AngelaFowler82 to confirm and close out!

**Comment by AngelaFowler82** [comment]
Headings are now nested properly. Officially closing.
